config.go

```diff
@@ -354,6 +354,8 @@ type Config struct {
 
 	GcCanceledInvoicesOnTheFly bool `long:"gc-canceled-invoices-on-the-fly" description:"If true, we'll delete newly canceled invoices on the fly."`
 
+	DustThreshold uint64 `long:"dust-threshold" description:"Sets the dust sum threshold in satoshis for a channel after which dust HTLC's will be failed."`
```

Comment:
re question in PR description: I think that it's fine to start with a single threshold and await a feature request that asks for per-channel (if it ever arrives).

Reply:
cool 👍